README.md

```diff
@@ -51,6 +51,9 @@ Once all the tools are installed, there is no need to invoke them directly, ever
 $ make codegen
 ```
 
+However, there are some deviations from the other SDKs, in which a manual code change is needed:
+- user agent parsing does not match for the Mac OS X operating system
+
```

Comment:
Can you also please add a section in here describing the need to pull the test files from the target-sdk-testing repo (include link).  And then mention that because this SDK does not support the telemetry feature we need to manually modify the TEST_SUITE_TELEMETRY.json file (include link to this commit).